api/peripheral_api/routes/Speaker.js

```diff
@@ -0,0 +1,80 @@
+const express = require('express');
+const axios = require('axios');
+const router = express.Router();
+const { speakerQueued } = require('../util/Speaker.js');
+const {
+  verifyToken,
+  checkIfTokenSent,
+} = require('../../util/token-verification');
+const {
+  OK,
+  BAD_REQUEST,
+  UNAUTHORIZED,
+  FORBIDDEN,
+  NOT_FOUND,
+  SERVER_ERROR,
+} = require('../../util/constants').STATUS_CODES;
+const logger = require('../../util/logger');
+
+router.post('/stream', async (req, res) => {
+  if (!checkIfTokenSent(req)) {
+    logger.warn('/updateSignText was requested without a token');
```

Comment:
`updateSignText` was requested without a token? fix this and the other  logs in this file